system/Entity/Entity.php

```diff
@@ -274,7 +274,7 @@ public function hasChanged(?string $key = null): bool
      *
      * @return $this
      */
-    public function setAttributes(array $data)
+    public function injectRawData(array $data)
```

Comment:
removing public function is BC, isn't it?

Reply:
Yes. I added `beraking change` label.